DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -460,6 +460,12 @@ public string BannerUrl
         [JsonProperty("premium_subscription_count", NullValueHandling = NullValueHandling.Ignore)]
         public int? PremiumSubscriptionCount { get; internal set; }
 
+        /// <summary>
+        /// Whether the guild has the boost progress bar enabled.
+        /// </summary>
+        [JsonProperty("premium_progress_bar_enabled", NullValueHandling = NullValueHandling.Include)]
```

Comment:
NVH include won’t do anything on a non-nullable struct :b

Reply:
Whoops, that's left over from debugging. Was meant to be `Ignore` as it was on the `NSFW` property

Reply:
Fixed in latest commit